source/_posts/2017-03-30-Haptic-feedback-makes-you-vibrate.md

```diff
@@ -0,0 +1,147 @@
+---
+title:  Haptic Feedback makes you vibrate!
+date: 2017-03-30 10:10:57
+tags: ios,haptic,feedback,vibrate
+authorIds: 
+- maco
+categories:
+- iOS
+---
+
+The iPhones could vibrate ever since they were launched. And so did almost all the other phones, even from before smartphones were a thing. But Apple took this function further than anyone else, by introducing the Taptic Engine in the Apple Watch, and after on in many other products, like the new MacBooks and iPhones. The Taptic Engine provides a haptic feedback, more similar to real-life touches. 
+
+Together with iOS 10, Apple introduced an API for the Taptic Engine, which the developers can use to give haptic feedback to their users. To fully experience the haptic feedback you need an iPhone 7, 7 Plus or newer. 
+
+## How to create haptic feedback
+
+Using the new API is super easy, with the `UIFeedbackGenerator` you can create haptic feedback with only two lines of code.
+
+Even though the API is introduced in iOS 10, it's only the iPhone 7 and iPhone 7 Plus that support this. On all previous devices, nothing happens when you try to do that. 
```

Comment:
I would maybe move this line up with the "Together with iOS 10" line, because they're related to eachother.